.env

```
@@ -4,19 +4,21 @@
 BITCOIN_ELECTRUM_API_MAINNET=18.217.213.66:50001
 BITCOIN_ELECTRUM_API_TESTNET=18.217.213.66:60001
 BITCOIN_ELECTRUM_API_SIGNET=18.217.213.66:60601
-BITCOIN_ELECTRUM_API_REGTEST=localhost:50001
+BITCOIN_ELECTRUM_API_REGTEST=18.217.213.66:60001
 BITCOIN_EXPLORER_API_MAINNET=https://mempool.space/api
 BITCOIN_EXPLORER_API_TESTNET=https://mempool.space/testnet/api
 BITCOIN_EXPLORER_API_SIGNET=https://mempool.space/signet/api
-BITCOIN_EXPLORER_API_REGTEST=http://localhost:3000/regtest/api
+BITCOIN_EXPLORER_API_REGTEST=https://mempool.space/testnet/api
 # BITCOIN_EXPLORER_API_MAINNET=http://127.0.0.1:3000
 # BITCOIN_EXPLORER_API_TESTNET=http://127.0.0.1:3000
 # BITCOIN_EXPLORER_API_SIGNET=http://127.0.0.1:3000
 # BITCOIN_ELECTRUM_API_MAINNET=mempool.space:50001
 # BITCOIN_ELECTRUM_API_TESTNET=mempool.space:60001
 # BITCOIN_ELECTRUM_API_SIGNET=mempool.space:60601
-NODE_HOST=https://qvijq4x0ei.execute-api.us-east-2.amazonaws.com/dev
+NODE_HOST=://hut8-ca-on-mississauga.carbonado.io/carbonadoapi.us-east-2.amazonaws.com/dev
 # NODE_HOST=http://127.0.0.1:7070
 LNDHUB_ENDPOINT=https://lndhubx.bitmask.app
 CARBONADO_ENDPOINT=http://localhost:7070/carbonado
-BITCOIN_NETWORK=regtest
+BITCOIN_NETWORK=bitcoin
```

I'm not sure, but in most places Network conversion expects mainnet not bitcoin.

```suggestion
BITCOIN_NETWORK=mainnet
```

---

I updated .env by mistake...